st2common/st2common/services/policies.py

```diff
@@ -24,6 +24,20 @@
 LOG = logging.getLogger(__name__)
 
 
+def has_policies(lv_ac_db, policy_types=None):
+    query_params = {
+        'resource_ref': lv_ac_db.action,
+        'enabled': True
+    }
+
+    if policy_types:
+        query_params['policy_type__in'] = policy_types
+
+    policy_dbs = pc_db_access.Policy.query(**query_params)
```

Comment:
Adding ``.count()`` to the end would probably be a bit more efficient since the count will be calculated and returned server side and means we don't need to evaluate and load the whole result set in memory like we do if we use ``len()``.

Not a huge issue here since those documents are not large, but still an easy change :)

Reply:
Fixed